crates/nu-command/src/conversions/into/filesize.rs

```diff
@@ -22,6 +22,22 @@ impl Command for SubCommand {
                 (Type::String, Type::Filesize),
                 (Type::Filesize, Type::Filesize),
                 (Type::Table(vec![]), Type::Table(vec![])),
+                (
+                    Type::List(Box::new(Type::Int)),
+                    Type::List(Box::new(Type::Filesize)),
+                ),
+                (
+                    Type::List(Box::new(Type::Number)),
+                    Type::List(Box::new(Type::Filesize)),
+                ),
+                (
+                    Type::List(Box::new(Type::String)),
+                    Type::List(Box::new(Type::Filesize)),
+                ),
+                (
+                    Type::List(Box::new(Type::Filesize)),
+                    Type::List(Box::new(Type::Filesize)),
+                ),
```

Comment:
Relax for heterogeneous list

Reply:
Fixed with [`ce063d5` (#9742)](https://github.com/nushell/nushell/pull/9742/commits/ce063d55e3db74eb3dbb63f32fb48b6dc73451c1)